package.json

```diff
@@ -100,7 +100,7 @@
     "chalk": "^2.3.0",
     "common-tags": "^1.7.2",
     "dag-map": "^2.0.2",
-    "ember-cli": "2.18.0",
+    "ember-cli": "github:ember-cli/ember-cli",
```

Comment:
do we want to keep this?

Reply:
no, ideally we'll update this to the next ember cli beta

Reply:
sorry I read what was written but didn't understand properly.
the work looks good :smiley: